src/case-monitoring/sub-process.ts

```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 import {type BpmnVisualization} from 'bpmn-visualization';
 import {type Instance} from 'tippy.js';
-import {Notyf} from 'notyf';
 import {displayView, isSubProcessBpmnDiagramIsAlreadyLoad, subProcessBpmnVisualization, subProcessViewName} from '../diagram.js';
+import {configureToast, NotyfType, toast} from '../utils/shared.js';
 import {AbstractCaseMonitoring, AbstractTippySupport} from './abstract.js';
 import 'notyf/notyf.min.css';
```

Comment:
**thought:** in the future, let's move the CSS import in a dedicated file. Notyf is not only used here, so we cannot place the CSS reference here.
If possible, do not use the minified version and let Vite manage the whole bundling/minification of all CSS files.
The